mmv1/products/compute/ServiceAttachment.yaml

```diff
@@ -156,6 +156,22 @@ properties:
           description: |
             The status of the connection from the consumer forwarding rule to
             this service attachment.
+        - !ruby/object:Api::Type::String
```

Comment:
It looks like the test is only running in the beta provider, should all these fields be `min_version: beta`?

Reply:
Those two fields are already available in the production API, but I will move the test checks of those fields to the update test to make sure they are tested in the GA provider.